pre-commit

```diff
@@ -10,6 +10,8 @@ if [ -L "$0" ]; then
 	else
 		DEV_LIB_PATH=$( dirname $( readlink "$0" ) )
 	fi
+elif [ -e "$(dirname "$0")/.dev-lib-path" ]; then
+    DEV_LIB_PATH="$(realpath $(cat "$(dirname "$0")/.dev-lib-path"))"
 else
```

Comment:
No, not using it, I'm on Mac but have a long Windows history 

There's only so many pull requests on championing of the following repo that one can do in their lifetime, but `scoop` is for Windows what `homebrew` is to Mac https://github.com/lukesampson/scoop, it's fantastic but also PowerShell focused rather than `cmd.exe`.

They have a symlink wrapper function that works perfectly, again this is PowerShell only though https://raw.githubusercontent.com/lukesampson/psutils/4e363db/ln.ps1

Try to find something that works for PowerShell, cmd.exe, and now bash on Windows I doubt exists